util-cache-guava/src/test/scala/com/twitter/cache/guava/LoadingFutureCacheTest.scala

```diff
@@ -4,7 +4,7 @@ import com.google.common.cache.{CacheLoader, CacheBuilder}
 import com.twitter.cache.AbstractLoadingFutureCacheTest
 import com.twitter.util.Future
 import org.junit.runner.RunWith
-import org.scalatest.junit.JUnitRunner
+import org.scalatestplus.junit.JUnitRunner
```

Comment:
We can actually remove this along with 
`org.junit.runner.RunWith`
and 
`@RunWith(classOf[JUnitRunner])`
but we will still need to update other things in this package like
`org.scalatestplus.mockito.MockitoSugar`

Reply:
Do you want to remove `@RunWith(classOf[JUnitRunner])` globally?

Reply:
We do, but should do it in another effort. I think we can remove these either before this change or after (assuming less conflicts for you)?

Reply:
If we remove before this PR, it will significantly reduce the number of files touched in this PR and with that its signal to noise ratio. The rebase against that will be big but easy for this PR. Happy to do that. WDYT?

Reply:
I think that sounds perfect!

Reply:
PR for that is here: https://github.com/twitter/util/pull/249

If CI is green there (and actually runs all tests), and can be merged, we can rebase this and reduce the diff size a notch or two.

Reply:
I've rebased on top of that one. The commit history of this PR is rather messy, and I'd like to rebase and squash into fewer commits to simplify review. If you want, we can split this PR in a couple of different ones along the same way we did this one:

* Update build file and packages without adding a 2.13 build
* Fixes of `Unit` used accidentally as `()`
* Update scalatestplus deprecated aliases
* Add the 2.13 build, and all other changes
* Potentially, fix the deluge of collection converter deprecation warnings when collectionCompat 1.2.0 is released (any day/hour now)